components/Card.js

```diff
@@ -19,4 +19,15 @@ export default function Card(cardElement) {
     }
     answerText.classList.toggle('card__answer--hide');
   });
+
+  const apiUrl = 'https://jservice.io/api/random';
```

Comment:
Maybe you would like to work with the OPENTRIVIA DB, where you will find real quiz questions.

Reply:
Thanks for the hint :)